tests/acceptance/infinity-load-previous-test.js

```diff
@@ -59,7 +59,7 @@ module('Acceptance: Infinity Route - load previous', function(hooks) {
 
     await triggerEvent(window, 'scroll');
 
-    shouldBeItemsOnTheList(assert, 104);
+    shouldBeItemsOnTheList(assert, 50);
```

Comment:
This was a bug that was fixed by moving the recursive previous page fetching function to the infinity-loader component instead of the service